example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -54,8 +55,26 @@ const s3data = [
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf'
 ];
 
-function isExecutionForGranuleId(taskInput, params) {
-  return taskInput.payload.granules && taskInput.payload.granules[0].granuleId === params.granuleId;
+function isReingestExecution(taskInput) {
```

Comment:
These functions should probably be extracted out to the helper files.

Reply:
Done.